app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -17,144 +17,112 @@
     android:layout_height="match_parent">
 
     <ScrollView
-      android:id="@+id/scroll"
+      android:id="@+id/add_profile_activity_scroll_view"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
       android:background="@color/addProfileBackground">
 
       <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:paddingTop="40dp"
-        android:paddingBottom="40dp">
-
-        <de.hdodenhof.circleimageview.CircleImageView
-          android:id="@+id/upload_image_button"
-          android:layout_width="136dp"
-          android:layout_height="136dp"
-          android:paddingBottom="8dp"
+        android:paddingTop="48dp"
+        android:paddingBottom="64dp">
+
+        <com.jackandphantom.circularimageview.CircleImage
+          android:id="@+id/add_profile_activity_user_image_view"
+          android:layout_width="120dp"
+          android:layout_height="120dp"
+          android:clickable="true"
+          android:focusable="true"
           android:src="@drawable/ic_default_avatar"
-          app:civ_circle_background_color="@color/grey"
+          app:add_shadow="true"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
+          app:layout_constraintTop_toTopOf="parent"
+          app:shadow_radius="8" />
 
-        <com.google.android.material.floatingactionbutton.FloatingActionButton
-          android:id="@+id/edit_image_fab"
-          android:layout_width="wrap_content"
+        <ImageView
+          android:id="@+id/add_profile_activity_edit_user_image_view"
+          android:layout_width="48dp"
+          android:layout_height="48dp"
+          android:paddingStart="12dp"
+          android:paddingTop="12dp"
+          android:src="@drawable/rounded_white_background_with_shadow"
+          app:layout_constraintBottom_toBottomOf="@+id/add_profile_activity_user_image_view"
+          app:layout_constraintEnd_toEndOf="@+id/add_profile_activity_user_image_view"
+          app:layout_constraintRight_toRightOf="@+id/add_profile_activity_user_image_view" />
+
+        <TextView
+          android:id="@+id/add_profile_activity_required_heading_text_view"
+          android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:src="@drawable/ic_edit"
-          app:backgroundTint="@color/white"
-          app:fabSize="mini"
-          app:layout_constraintBottom_toBottomOf="@+id/upload_image_button"
-          app:layout_constraintEnd_toEndOf="@+id/upload_image_button" />
+          android:layout_marginStart="32dp"
```

Comment:
Margin start is 28dp in the  mock, please correct it.

Reply:
Done.